### ISSUE TITLE
Font Family Issue

### ISSUE THREAD

**Issue by manishbisht** [entria/react-native-fontawesome#65]
iOS uses the `PostScript name` of the font. The Postscript name is changed in the Font Awesome 5.9 and the PostScript name is now `FontAwesome5Free-Solid`

<img width="794" alt="Screen Shot 2019-08-07 at 11 16 16 AM" src="https://user-images.githubusercontent.com/14255990/62597935-38c63680-b905-11e9-8556-53d7eac1040e.png">
<img width="709" alt="Screen Shot 2019-08-07 at 11 16 44 AM" src="https://user-images.githubusercontent.com/14255990/62597945-3cf25400-b905-11e9-9b85-c3d5e9c59216.png">
<img width="390" alt="Screen Shot 2019-08-07 at 11 18 46 AM" src="https://user-images.githubusercontent.com/14255990/62597947-3f54ae00-b905-11e9-8f18-ad78a431b603.png">

**Comment by manishbisht** [comment]
If this issue gets confirmed. I can submit the fix and send the PR.

**Comment by leandrosimoes** [comment]
> If this issue gets confirmed. I can submit the fix and send the PR.

Hi @manishbisht ! Try to use [this](https://github.com/leandrosimoes/react-native-fontawesome/tree/update-sample-project-react-native-version/sample/assets/fonts) font files there is in the new `sample/` project that I made. This files match the font family names. I changed the downlod url in the documentation, but this was not merged yet as you can see at this PR: https://github.com/entria/react-native-fontawesome/pull/64

**Comment by sspiff** [comment]
Changing the fontFamily like so is an easy workaround:

    import FontAwesome, { SolidIcons } from 'react-native-fontawesome'
    SolidIcons._fontFamily = 'FontAwesome5Free-Solid'

**Comment by leandrosimoes** [comment]
> Changing the fontFamily like so is an easy workaround:
> 
> 
> ```
> import FontAwesome, { SolidIcons } from 'react-native-fontawesome'
> SolidIcons._fontFamily = 'FontAwesome5Free-Solid'
> ```

Did you try to use [this](https://github.com/entria/react-native-fontawesome/tree/master/sample/assets/fonts) font file? Is from the sample project and the sample project was tested in both, android and ios, and it is all working.

**Comment by sspiff** [comment]
> > Changing the fontFamily like so is an easy workaround:
> > ```
> > import FontAwesome, { SolidIcons } from 'react-native-fontawesome'
> > SolidIcons._fontFamily = 'FontAwesome5Free-Solid'
> > ```
> 
> Did you try to use [this](https://github.com/entria/react-native-fontawesome/tree/master/sample/assets/fonts) font file? The files are from the sample project and the sample project was tested in both, android and ios, and it is all working.

No, I had followed the link to the font files in the (outdated?) README on npmjs.com (which point to fontawesome.com).

**Comment by leandrosimoes** [comment]
> > > Changing the fontFamily like so is an easy workaround:
> > > 
> > >   
> > >     
> > >       
> > >     
> > >   ```
> > > import FontAwesome, { SolidIcons } from 'react-native-fontawesome'
> > > SolidIcons._fontFamily = 'FontAwesome5Free-Solid'
> > > ```
> > 
> > 
> > Did you try to use [this](https://github.com/entria/react-native-fontawesome/tree/master/sample/assets/fonts) font file? The files are from the sample project and the sample project was tested in both, android and ios, and it is all working.
> 
> No, I had followed the link to the font files in the (outdated?) README on npmjs.com (which point to fontawesome.com).

Yeah, the npm README is outdated, unfortunately. I'll open an Issue to update that asap. Try to use the font files from the link that I told you, I think that it will work.

**Comment by vicnicius** [comment]
The suggested fix will only work for the free version of the font, though. If using Pro - like I was trying to -, one will likely have to come up with a different workaround. :/
